examples/yarn-pnp/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "remix-example-yarn-pnp",
+  "private": true,
+  "description": "",
+  "license": "",
+  "sideEffects": false,
+  "scripts": {
+    "build": "remix build",
+    "dev": "remix dev",
+    "start": "remix-serve build"
+  },
+  "dependencies": {
+    "@remix-run/node": "1.5.1",
+    "@remix-run/react": "1.5.1",
+    "@remix-run/serve": "1.5.1",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "~": "link:./app"
```

Comment:
What does this line do? I was expecting to see an `import {...} from "~/...";` somewhere in the example codebase that exercises this, but maybe I'm misunderstanding the purpose of `~` and `link:`

Reply:
PnP doesn't know or care about `tsconfig.json` files. In order to use the `~` alias that the other examples all provide through their `tsconfig.json`, we can use the `link:` protocol that yarn supports to remedy this issue. (See also https://yarnpkg.com/features/protocols#why-is-the-link-protocol-recommended-over-aliases-for-path-mapping)

Reply:
Ok that makes sense, but I still can't find anywhere in the `examples/yarn-pnp` code where the `~` is actually used.

If its not used at all, could we remove this alias from the example to keep things minimal?